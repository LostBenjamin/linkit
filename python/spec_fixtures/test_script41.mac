
$num-outer 5
$num-inner 5

[app]
(((do-outer <num-outer>)))

(((outer <num-outer>)))

[[do-outer INSTANCE
    (doit-INSTANCE)
]]

[[do-inner INSTANCE PARENT
    (doit-PARENT-INSTANCE)
]]

[[outer INSTANCE
   [doit-INSTANCE]
       (((do-inner <num-inner> INSTANCE)))

   (((inner <num-inner> INSTANCE)))
]]

[[inner INSTANCE PARENT
    [doit-PARENT-INSTANCE]
        INSTANCE
        red
        PARENT
        blue
]]

