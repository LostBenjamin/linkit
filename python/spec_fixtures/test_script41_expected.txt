4:set:6:run:13:run:20:run:27:run:34:run
6:set:8:run:9:run:10:run:11:run:12:run
8:set:0:red:0:blu
9:set:1:red:0:blu
10:set:2:red:0:blu
11:set:3:red:0:blu
12:set:4:red:0:blu
13:set:15:run:16:run:17:run:18:run:19:run
15:set:0:red:1:blu
16:set:1:red:1:blu
17:set:2:red:1:blu
18:set:3:red:1:blu
19:set:4:red:1:blu
20:set:22:run:23:run:24:run:25:run:26:run
22:set:0:red:2:blu
23:set:1:red:2:blu
24:set:2:red:2:blu
25:set:3:red:2:blu
26:set:4:red:2:blu
27:set:29:run:30:run:31:run:32:run:33:run
29:set:0:red:3:blu
30:set:1:red:3:blu
31:set:2:red:3:blu
32:set:3:red:3:blu
33:set:4:red:3:blu
34:set:36:run:37:run:38:run:39:run:40:run
36:set:0:red:4:blu
37:set:1:red:4:blu
38:set:2:red:4:blu
39:set:3:red:4:blu
40:set:4:red:4:blu

